js/index.js

```diff
@@ -0,0 +1,136 @@
+const getSpeakersInfo = () => ({
+  speaker1: {
+    name: 'Jensen Huang',
+    postion: 'Founder and CEO, NVIDIA',
+    description: 'GTC 2022 Keynote',
+  },
+  speaker2: {
+    name: 'Chelsea Finn',
+    postion: 'Assistant Professor, Stanford University',
+    description: 'Scaling Data-Driven Robot Learning',
+  },
+  speaker3: {
+    name: 'Serdar Kadioglu',
+    postion: 'Vice President AI, Fidelity Investments | Brown University',
+    description: 'Multi-Objective Optimization to Boost Exploration in Recommender Systems',
+  },
+  speaker4: {
+    name: 'Khalifeh AlJadda',
+    postion: 'Sr. Director of Data Science, The Home Depot',
+    description: 'Building AI-based Recommender System Leveraging the Power of Deep Learning and GPU',
+  },
+  speaker5: {
+    name: 'Xiaodi Hou',
+    postion: 'Co-Founder and CTO, TuSimple',
+    description: 'Achieving Level 4 Autonomy at Scale: How TuSimple is Building on NVIDIA DRIVE Orin to Deploy',
+  },
+  speaker6: {
+    name: 'Dale Durran',
+    postion: 'Professor, University of Washington, Department of Atmospheric Sciences',
+    description: 'Bringing Rain to the Subseasonal Forecasting Desert with Deep Learning Weather Prediction',
+  },
+});
+const createSpeakersSection = () => {
+  const speakersInfo = getSpeakersInfo();
+  const speakersSection = document.querySelector('#speakers');
+  const heading = document.createElement('h2');
+  heading.id = 'speakers-heading';
+  heading.innerText = 'Featured Speakers';
+  speakersSection.appendChild(heading);
+  const hr = document.createElement('hr');
+  speakersSection.appendChild(hr);
+  const ul = document.createElement('ul');
+  for (let i = 0; i < 6; i += 1) {
+    const li = document.createElement('li');
+    const speaker = document.createElement('img');
+    speaker.alt = `speaker ${i + 1}`;
```

Comment:
- [ ] [OPTIONAL] Please see my comments about avoiding hard coding index values. you should avoid hard coming the value `6` because thinking about a situation where you will have more than six speakers you would have to change this number in several places.